edgelet/edgelet-utils/src/lib.rs

```diff
@@ -80,22 +80,3 @@ pub fn prepare_dns_san_entries<'a>(
         }
     })
 }
-
-pub fn append_dns_san_entries(sans: &str, names: &[&str]) -> String {
```

Comment:
Was this never used anywhere?

Reply:
it used to be used, but then after some refactor a different code took over that functionality (the one changed by this PR).